lib/telephone_appointments/stub_api.rb

```diff
@@ -1,4 +1,7 @@
 module TelephoneAppointments
   class StubApi
+    def post(_path, _form_data)
```

Comment:
I'd just favour `post(*)` as the arguments are inconsequential really and the arity isn't worth preserving.